.envrc

```diff
@@ -0,0 +1 @@
+use flake . --impure
```

Comment:
`.envrc` files generally shouldn't be committed in git, as different users may want to set their direnvs differently.
Can this be reflected in the build instructions instead?

Reply:
Gotcha, I'll take it out then.